src/components/NavBar.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './NavBar.css';
+import { Link } from 'react-router-dom';
+import user from '../images/user.png';
+
+function Nav() {
+  return (
+    <div className="divnav commun-divnav">
+      <div className="DivNav commun-divnav">
+        <div className="Bookstore-CMS commun-style">Bookstore CMS </div>
+        <div className="BOOKS commun-nav"><Link to="./" className="book-link">Book</Link></div>
```

Comment:
Your design
![image](https://github.com/sam2if/Bookstore/assets/78251927/5acb1438-9815-4c71-8d27-2f7393e7beeb)

Template
![image](https://github.com/sam2if/Bookstore/assets/78251927/cef06a21-72bb-4c27-8013-4b665196aee6)

Please adjust the layout of the title `Bookstore CMS` to match the template